js/test.js

```diff
@@ -0,0 +1,4 @@
+var
+
+var
```

Comment:
Expected an identifier and instead saw 'var' (a reserved word).<br>Missing semicolon.<br>'var' is defined but never used.